fe/fe-core/src/main/java/org/apache/doris/planner/RepeatNode.java

```diff
@@ -131,7 +131,8 @@ public void init(Analyzer analyzer) throws UserException {
         // build tupleDesc according to child's tupleDesc info
         outputTupleDesc = groupingInfo.getVirtualTuple();
         //set aggregate nullable
-        for (Expr slot : groupByClause.getGroupingExprs()) {
+        for (int i = 0; i < groupByClause.getGroupingExprs().size() - 1; i++) {
```

Comment:
It doesn't look like it needs to be changed?

Reply:
The last grouping expr is virtual grouping id， not need to set nullable